index.html

```diff
@@ -2,52 +2,149 @@
+      <p>Cumque provident nam, inventore repudiandae animi doloribus dignissimos illo ea error esse reprehenderit possimus odio, libero beatae, accusamus ratione itaque laboriosam expedita amet nihil voluptatum blanditiis obcaecati. Possimus, assumenda sint.</p>
+      <p>Sunt quas dicta, vel perspiciatis ducimus placeat voluptates ad necessitatibus quis reprehenderit ullam consectetur! Enim neque ab architecto praesentium voluptatem saepe commodi, aperiam odio facere illum incidunt laborum laudantium pariatur?</p>
+      <p>Minima quo molestias asperiores nam nisi officia impedit molestiae, sed hic reiciendis. Officia atque labore odit aliquid inventore ab possimus, incidunt provident ut at, neque consequatur minus quod expedita eligendi!</p>
+      <p>Amet unde eligendi provident molestias suscipit itaque exercitationem voluptatum, ut natus id quas consectetur illo! Libero cum eum sit autem aliquid officia consequuntur tempore velit rerum? Vero blanditiis odio omnis.</p>
+    </div>
+    <div class="deck">
+      <div class="card">
+        <i class="material-icons" onclick="window.open('/form.php?quid=511','_self')">email</i>
+        <div class="inline">
+          <h3><a href="/form.php?quid=511">Email</a></h3>
+          <p>Email us your research questions and we&rsquo;ll respond within 24 hours.</p>
+        </div>
+      </div>
+  
+      <div class="card">
+        <i class="material-icons" onclick="document.querySelector('button.s-lch-widget-float-btn').click();">question_answer</i>  
+        <div class="inline">
+          <h3><a onclick="console.log('chat says hi')" href="javascript:void(0)">Chat</a></h3>
+          <p>Talk online to a research librarian on weekdays and Sundays</p>
+        </div>
+      </div>
+  
+      <div class="card">
+        <i class="material-icons" onclick="window.open('tel:6173532700','_self')">phone</i>
+        <div class="inline">
+          <h3><a href="tel:6173532700">Call</a></h3>
+          <p>Call us at 617-353-2700 during our open hours</p>
+        </div>
+      </div>
+    </div>
+    <div class="ctas">
+      <div class="cta" onclick="window.open('/search/?g=716&amp;topics=Borrowing','_self')">
+        <h3><a href="/search/?g=716&amp;topics=Borrowing">Borrowing Items</a></h3>
+      </div>
+  
+      <div class="cta" onclick="window.open('/search/?t=0&amp;g=716&amp;topics=Alumni&amp;adv=1','_self')">
+        <h3><a href="/search/?t=0&amp;g=716&amp;topics=Alumni&amp;adv=1">Alumni</a></h3>
+      </div>
+  
+      <div class="cta" onclick="window.open('/search/?t=0&amp;g=716&amp;topics=Where%20Is%20It%3F&amp;adv=1','_self')">
+        <h3><a href="/search/?t=0&amp;g=716&amp;topics=Where%20Is%20It%3F&amp;adv=1">Where is it?</a></h3>
+      </div>
+  
+      <div class="cta" onclick="window.open('/businessFAQs/','_self')">
+        <h3><a href="/businessFAQs/">Pardee / Management</a></h3>
+      </div>
+    </div>
     <bulib-footer></bulib-footer>
```

Comment:
note: web components aren't loading correctly when this file is served with `es-dev-server`. we're getting an "Uncaught TypeError", but it's an [issue for another time](https://bulibrary.atlassian.net/browse/WEB-268).